README.md

```diff
@@ -1,69 +0,0 @@
-<img src="./sally.svg" width="50"/>
```

Comment:
Maybe it. would be good to keep the README.md for the github repo, with a link to CONTRIBUTING.md for example

Reply:
Somethinke like

```
# awesome-badger

Amazing blog posts posted by amazing badgers.

See CONTRIBUTING if you want to add something.
```

Reply:
Yup, just adding that now...